index.html

```diff
@@ -1273,7 +1273,26 @@ <h3>Query By Example</h3>
   "domain": "reunion.example"
 }
         </pre>
-
+        
+        <p class="note"
+         title="Selective disclosure using Query By Example">
+          Any field included in a Query By Example is a required field. This means that to use a Query By Example 
+          to enable selective disclosure, a requester should be mindful not to include fields in their query that 
+          are not needed for their use case. Query By Example provides no way to indicate an "optional" field,  
+          consequently, it is up to the wallet that is responding to the query to ensure its response does not 
+          reveal additional information beyond what is being requested. Note: Some securing mechanisms
+          for verifiable credentials do not permit selective disclosure; in these cases even unrequested
+          information might have to be sent in order to send the requested information. Digital wallets are
+          expected to inform users about what information will be shared.
+
+          To request a field without any statement about an expected value, a requester can include the field in 
+          the request but leave the value as an empty string. For instance, if a requester needs a first name, 
+          they would include the optional "credentialSubject" object and only include a single field, "firstName": "". 
+          This would allow a wallet to respond with only the firstName of the requested credential with no expectation 
+          of any other field being included. This does not prevent a wallet from oversharing and including additional 
+          fields in its response to the query.
```

Comment:
```suggestion
          This would allow a wallet to respond with only the `firstName` of the requested credential with no expectation 
          of any other field being included. _This does not prevent a wallet from oversharing by including additional 
          fields in its response to the query._
```